examples/apps/simple_wallet/workload/simple_wallet_logic.cpp

```diff
@@ -0,0 +1,179 @@
+/* Copyright 2019 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include <string>
+#include <string.h>
+#include <stdint.h>
+#include <stdarg.h>
+#include <vector>
+
+#include "simple_wallet_logic.h"
+#include "simple_wallet_execute_io.h"
+#include "enclave_utils.h"
+
+bool IsValidNumber(std::string numStr) {
+    int num = 0;
+    try {
+        num = std::stoi(numStr);
+    } catch (...) {
+        return false;
+    }
+    return (num > 0) ? true : false;
+}
+
+// Create account identified by accountName if not created and
+// update wallet balance
+std::string DepositMoney(std::string accountName, std::string amount,
+    std::string encryptionKey) {
+    std::string walletBalance = ReadWalletBalance(accountName, encryptionKey);
+    if (walletBalance.empty()) {
+        walletBalance = UpdateWalletBalance(accountName, amount, encryptionKey);
+        Log(TCF_LOG_INFO, "Wallet Created for the account %s. Wallet Balance: %s",
+            accountName.c_str(), amount.c_str());
+    } else {
+        Log(TCF_LOG_INFO, "Wallet Balance before deposit: %s",
+            walletBalance.c_str());
+        std::string netWalletBalance = std::to_string(std::stoi(walletBalance) +
+            std::stoi(amount));
+        Log(TCF_LOG_INFO, "net Wallet Balance after deposit: %s",
+            netWalletBalance.c_str());
+        walletBalance = UpdateWalletBalance(accountName, netWalletBalance,
+            encryptionKey);
+        Log(TCF_LOG_INFO, "Updated Wallet Balance: %s",
+            walletBalance.c_str());
```

Comment:
Why do we need to log both netWalletBalance and Updated Wallet Balance ? Are both different ? If not, can we just log one of it.

Reply:
log to print netWalletBalance was added for debugging purpose, will remove it.